rdr_service/model/study_nph.py

```diff
@@ -279,3 +282,21 @@ class SampleExport(NphBase):
     ignore_flag = Column(TINYINT, default=0)
     export_id = Column(BigInteger, ForeignKey("biobank_file_export.id"))
     sample_update_id = Column(BigInteger, ForeignKey("sample_update.id"))
+
+
+class StoredSample(NphBase):
+    __tablename__ = "stored_sample"
+
+    id = Column("id", BigInteger, autoincrement=True, primary_key=True)
+    created = Column(UTCDateTime)
+    modified = Column(UTCDateTime)
+    biobank_modified = Column(UTCDateTime)
+    biobank_id = Column(BigInteger, ForeignKey("participant.biobank_id"))
+    sample_id = Column(BigInteger)
```

Comment:
Can you set `index=True` on the `sample_id`? I have a feeling we'll need it in the long run.